blockchain/stakeversion_test.go

```diff
@@ -124,6 +128,7 @@ func TestCalcStakeVersionCorners(t *testing.T) {
 	bc := &BlockChain{
 		chainParams: params,
 	}
+	bc.initializeStakeVersionCaches()
```

Comment:
This would be much cleaner and more to have a `newFakeChain` function which returns the `BlockChain` instance with all of the maps already initialized in the definition.

Then all of these repeated

```Go
bc := &BlockChain{
	chainParams: params,
}
bc.initializeStakeVersionCaches()
```

Would just turn into:
```Go
bc := newFakeChain(params)
```